tools/AutomationTestFramework/ManageScripts/AccountRunbook.ps1

```diff
@@ -0,0 +1,102 @@
+if (!$azureConfig) {
```

Comment:
General comment, please add the license header as here:  https://github.com/Azure/azure-powershell/blob/preview/tools/PublishModules.ps1#L1 to the top of each of the script files

Reply:
done.